it/src/test/java/com/linecorp/centraldogma/it/mirror/git/GitMirrorTest.java

```diff
@@ -212,9 +212,65 @@ void remoteToLocal() throws Exception {
                                            Entry.ofText(rev4, "/final_fantasy_xv.txt", "29-Nov-2016\n"));
     }
 
+    @Test
+    void remoteToLocal_gitIgnore() throws Exception {
+        pushMirrorSettings(null, "/first", "/exclude_if_root.txt" + "\\n" + "exclude_dir");
```

Comment:
Could you add more test cases that the `localPath` and `remotePath` have sub-directories?
e.g.
- `localPath == /a` and `remotePath == /b/c`
- `localPath == /a/b` and `remotePath == /c`

Reply:
Q: By the tests, what cases will be covered additionally?
I'm a little confused because they're being covered by `remoteToLocal_subdirectory`

Reply:
Ah, exactly we don't need the case. 😅 
How about also adding a case that the remote repository has a new commit that only contains ignored files?